release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -432,6 +438,16 @@ submodule openconfig-bgp-common {
         "Time-to-live value to use when packets are sent to the
         referenced group or neighbors and ebgp-multihop is enabled";
     }
+
+    leaf preserve-next-hop {
+     type boolean;
+     default "false";
+     description
+      "Override the default eBGP behavior of rewriting the next-hop, and
+      instead keep the next-hop learned from a neighbor.  This leaf can
+      only be configured if multihop enabled is true.";
```

Comment:
The placement of this leaf and the multihop requirement are not correct. This only standardizes the limitation of a specific implementation. Fundamentally there's no tie between the two.

Reply:
Thanks for the feedback, moved to afi-safi hierarchy